public/config/README_configuration.md

```diff
@@ -79,10 +79,11 @@ the widget properties have the following sections:
 * data.topicType: (value, state, joystick) ROS message type of the topic
 * data.topicDirection: (value, state, joystick) ROS topic direction (pub/sub)
 * data.topicAttribute: (value, state) ROS message attribute to display
+* data.topicAttribute: (slider) ["angle","name"] position based properties for value,name
 * data.topicPeriodS: (joystick, slider) Only when data.topicDirection is "publish",
                                         the browser UI will continually publish the
                                         current value with a period of
-                                        data.topicPeriodS seconds
+                                        data.topicPeriodS seconds. won't fire if js "falsey"
```

Comment:
I think it would be more clear to say "if the values should only be sent once, either remove the data.topicPeriodS from the configuration or set its value to 0".